### PR TITLE
Waist bags

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Adds waist bags, a new belt-storage item. They have no whitelist and can take any item 20 size or under, but their small container size may make them undesirable compared to other alternatives.

Right now, the colored waist bags do not exist since worn items having the color component is broken, and adding a unique texture for every single colored waist bag would be absurd. The textures for the parts of the colored waist bag exist for when it is fixed, though.

**Media**

Content Client_O8kHIcEoQP

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added waist bags, get them from your nearest clothesmate vendor today!

### CODE REVIEW

**Comment by vulppine** [comment]
make it so you can wear these on your back for the drip 😎 

(you don't have to, unless you can sprite that and you want to)

**Comment by metalgearsloth** [comment]
Please don't call it fannypack lmao.

**Review by vulppine — CHANGES_REQUESTED** [review]
just rename this to waist bag and you should be OK unless there's some balance issue against these (i remember ss13 having these with no real issue, imo)

**Comment by EmoGarbage404** [comment]
they're called fanny packs??? idk what y'all are on about.
(also i've always seen it written as 2 words: "fanny pack")

**Comment by Nylux** [comment]
> they're called fanny packs??? idk what y'all are on about. (also i've always seen it written as 2 words: "fanny pack")

I think they mean because of the slang use ?
https://www.urbandictionary.com/define.php?term=Fanny

**Comment by ChilbroBaggins** [comment]
Where in the world do people call these waist bags?

**Comment by vulppine** [comment]
> Where in the world do people call these waist bags?

Screenshot_20230310_174341

**Comment by ChilbroBaggins** [comment]
Wild. I've never heard any term other than fanny pack before.

**Comment by Flareguy** [comment]
Weird, I've never heard them called waist bags before. I do guess it makes it more clear what it is anyways, so I'm not exactly opposed.

**Comment by Emisse** [comment]
pussy pouch and everyone wins

File: Resources/Prototypes/Entities/Clothing/Belt/fannypacks.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+- type: entity
```
Comment (vulppine) [inline Resources/Prototypes/Entities/Clothing/Belt/fannypacks.yml:1]: looks like this is still here

**Review by vulppine — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/Belt/fannypacks.yml (hunk 1 above)
Comment (Flareguy) [reply Resources/Prototypes/Entities/Clothing/Belt/fannypacks.yml]: All fixed

File: Resources/Textures/Clothing/Belt/fannypack.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
```
Comment (vulppine) [inline Resources/Textures/Clothing/Belt/fannypack.rsi/meta.json:1]: this too

**Review by vulppine — CHANGES_REQUESTED** [review]

File: Resources/Textures/Clothing/Belt/fannypack.rsi/meta.json (hunk 2 above)
Comment (Flareguy) [reply Resources/Textures/Clothing/Belt/fannypack.rsi/meta.json:1]: I have genuinely no clue how I missed this. What the fuck

**Comment by deltanedas** [comment]
bikeshedding time
i also agree its fanny pack not fannypack or waist bag (though i do not wear supreme so maybe its just a class difference)

**Comment by vulppine** [comment]
> bikeshedding time i also agree its fanny pack not fannypack or waist bag (though i do not wear supreme so maybe its just a class difference)

this is mostly to appeal to the fact that 'fanny' is slang for 'pussy' in other parts of the world - i personally wouldn't mind either way (being from america, i've known it as a 'fannypack' for a while as well), but since there's a neutral alternative (using supreme as the example just happened to be the closest convenient thing i could find) i'm leaning towards that

**Review by vulppine — APPROVED** [review]
uh uhhhh uhhhhhhhhhhh

Screenshot_20230319_124445

**Comment by deltanedas** [comment]
babe wake up its time to change branch name to fanny-packs and make new pr

**Comment by PuroSlavKing** [comment]
oh no... it is not displayed in the hands...

**Comment by Flareguy** [comment]
> oh no... it is not displayed in the hands...

/tg/ didn't have inhands and i didn't feel like making them (also i fucking hate the clothing downscales and i refuse to make any myself)

**Pull request merged**